airflow/providers/google/cloud/hooks/dataform.py

```diff
@@ -263,9 +301,28 @@ def cancel_workflow_invocation(
             f"projects/{project_id}/locations/{region}/repositories/"
             f"{repository_id}/workflowInvocations/{workflow_invocation_id}"
         )
-        client.cancel_workflow_invocation(
-            request={"name": name}, retry=retry, timeout=timeout, metadata=metadata
-        )
+        try:
+            workflow_invocation = self.get_workflow_invocation(
+                project_id=project_id,
+                region=region,
+                repository_id=repository_id,
+                workflow_invocation_id=workflow_invocation_id,
+            )
+            state = workflow_invocation.state
+        except Exception as err:
+            self.log.info(
+                "Retrying. Dataform API returned error when waiting for workflow invocation: %s", err
+            )
```

Comment:
1. The error message says `Retrying` but there is no retry logic here. Perhaps, we shouldn't use this word here?
2. Instead of logging the error message with INFO level I would suggest throwing an `AirflowException` here because (1) errors should be visible, (2) if API raises exception, then the `state` variable remain unassigned, and it will raise another exception in the following conditional operator.

Reply:
Done